libs/openFrameworks/math/ofMatrix4x4.cpp

```diff
@@ -931,10 +931,10 @@ void ofMatrix4x4::makeLookAtMatrix(const ofVec3f& eye,const ofVec3f& center,cons
 	ofVec3f xaxis = up.getCrossed(zaxis).getNormalized();
 	ofVec3f yaxis = zaxis.getCrossed(xaxis);
 
-	_mat[0].set(xaxis.x, xaxis.y, xaxis.z, 0);
-	_mat[1].set(yaxis.x, yaxis.y, yaxis.z, 0);
-	_mat[2].set(zaxis.x, zaxis.y, zaxis.z, 0);
-	_mat[3] = eye;
```

Comment:
ofNode, which is used in lots of places, internally sets the lookAt like;

```
		ofMatrix4x4 m;
		m._mat[0].set(xaxis.x, xaxis.y, xaxis.z, 0);
		m._mat[1].set(yaxis.x, yaxis.y, yaxis.z, 0);
		m._mat[2].set(zaxis.x, zaxis.y, zaxis.z, 0);
```

so except for that 4th row i'd say is correct. the difference between lookAtMatrix and lookAtViewMatrix is that lookAtMatrix is supposed to be used for objects while lookAtViewMatrix is used to set up a view matrix for a camera which is the inverse